src/main/rx/fport.c

```diff
@@ -282,13 +282,14 @@ static uint8_t fportFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
                         switch(frame->data.telemetryData.frameId) {
                         case FPORT_FRAME_ID_NULL:
                         case FPORT_FRAME_ID_DATA: // never used
-                            hasTelemetryRequest = true;
+                            result = result | RX_FRAME_PROCESSING_REQUIRED;
```

Comment:
Should be `result |= RX_FRAME_PROCESSING_REQUIRED;`

Reply:
I'm not a fan of the combination operators (probably my upbringing in Oberon), but I can change it if you prefer.

Reply:
I love 'em. Besides they are widely used in betaflight, so there is the consistency argument...

Reply:
Why is that? They don't add any new capabilities, just inconsistency in how operations are expressed.

The long form and the short form seem to be both used fairly widely, so I don't think there is much of a consistency argument both ways.

Reply:
The reason I like them is they reflect the underlying meaning/intent. So `x *= 2;` precisely reflects the underlying meaning of "multiply x by 2", whereas `x = x*2;` does not. This is even more true of the increment/decrement operators `++x` means "increment x" and reflects that precisely, whereas `x = x+1;` is just long-winded.

Your argument against the combination operators equally applies to the increment/decrement operators, and then we'd be stuck with `for (i = 0; i<10; i = i+1)`

I also think the my argument applies strongly to the bitwise operators
`result |= RX_FRAME_PROCESSING_REQUIRED;` means set the `RX_FRAME_PROCESSING_REQUIRED` in `result` and precisely reflects that, whereas , in my view, `result = result | RX_FRAME_PROCESSING_REQUIRED;` obscures that meaning.

Reply:
Dunno. To me it is always an operation and an assignment. I don't see why 
```
x = x * 2
```
should be coded inconsistently to
```
x = y * 2
```
or
```
x = z * 2
```

But happy to change it if we want make it a coding standard that simple assignments have to be avoided where possible.